src/games/strategy/triplea/TripleA.java

```diff
@@ -79,93 +78,69 @@
   @Override
   public void shutDown() {
     super.shutDown();
-    if (m_display != null) {
-      game.removeDisplay(m_display);
-      m_display.shutDown();
-      m_display = null;
+    if (display != null) {
+      game.removeDisplay(display);
+      display.shutDown();
+      display = null;
     }
   }
 
-  protected void initializeGame() {}
 
   @Override
-  public void startGame(final IGame game, final Set<IGamePlayer> players, final boolean headless) throws Exception {
```

Comment:
The big try catch around the method is removed. Then the ad-hoc error handling is all removed as well since the library function now handles the error handling in a unified way. It was interesting to see the variations of how we handled Interrupted exception, sometime it was fully handled very formally, other time it was just ignored, and ranges in between.